File: ejercicios parte 1/1)programas simples/4conversion de unidades de logitud/conversion.cpp
#include <iostream>
using namespace std;

int main(){
    int cm;
    float in;
    cout<<"Ingrese la distancia en centimetros: ";
    cin>>cm;
    in = cm/2.54;
    cout<<"Tu distancia en pulgadas es: "<<in<<endl;
    return 0;
}

File: ejercicios parte 1/1)programas simples/4conversion de unidades de logitud/conversion.java
import java.util.Scanner;

public class conversion {

    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int cm;
            double in;

            System.out.println("Ingrese una distacia en cm: ");       
            cm = scan.nextInt();
            in= cm / 2.45;
            System.out.println("Esto es equivalente a "+ in +" pulgadas");
        }
    }
}